CONTRIBUTING.md

```diff
@@ -99,3 +99,34 @@ We believe in open source contributions as a learning experience – everyone is
 We use pair programming sessions as a tool for knowledge sharing. We can learn a lot from each other and only by sharing what we know and overcoming challenges together can we grow as a team and truly own this project collectively.
 
 As a volunteeer you have no commitment except your own self development and your awesomeness by contributing to this free and open-source software project. Cheers to you!
+
+
+## Open-Source Bounties
+
+There are so many good reasons to contribute to Human Connection
+* You learn state-of-the-art technologies
+* You build your portfolio
+* You contribute to a good cause
+
+Now there is one more good reason: You can receive a small fincancial
+compensation for your contribution! :tada:
+
+### How it works
+
+Before you can benefit from the Open-Source bounty program you **must get one
+pull request approved and merged for free**. You can choose something really
+quick and easy. What's important is starting a working relationship with the
+team, learning the workflow, and understanding this contribution guide. You can
+filter issues by 'good first issue', to get an idea where to start. Please join
+our our [community chat](https://human-connection.org/discord), too.
+
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-27T10:50:30Z" title="Monday, January 27th 2020, 11:50:30 am +01:00">Jan 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -99,3 +99,35 @@ We believe in open source contributions as a learning experience – everyone is
 We use pair programming sessions as a tool for knowledge sharing. We can learn a lot from each other and only by sharing what we know and overcoming challenges together can we grow as a team and truly own this project collectively.
 
 As a volunteeer you have no commitment except your own self development and your awesomeness by contributing to this free and open-source software project. Cheers to you!
+
+
+## Open-Source Bounties
+
+There are so many good reasons to contribute to Human Connection
+* You learn state-of-the-art technologies
+* You build your portfolio
+* You contribute to a good cause
+
+Now there is one more good reason: You can receive a small fincancial
+compensation for your contribution! :tada:
+
+### How it works
+
+Before you can benefit from the Open-Source bounty program you **must get one
+pull request approved and merged for free**. You can choose something really
+quick and easy, e.g. a fixing a typo in our [documentation](https://docs.human-connection.org/human-connection/)
+would be enough. But getting at least one pull request merged will make you get
+in touch with the team, learn the workflow and understand this contribution
+guide. Please join our our [community chat](https://human-connection.org/discord),
+too.
```


I question whether `fixing a typo` would be sufficient to get to know the "workflow and understand this contribution
guide." 
I'm in favor of it being a simple issue, but maybe it could be a little more technical?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-27T15:19:01Z" title="Monday, January 27th 2020, 4:19:01 pm +01:00">Jan 27, 2020</time>_

---


Sure, what is your wording here?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-27T16:36:38Z" title="Monday, January 27th 2020, 5:36:38 pm +01:00">Jan 27, 2020</time>_

---


I would say `easy, what's important is starting a working relationship with the team, learning the workflow, and understanding this contribution guide. You can filter issues by 'good first issue', to get an idea where to start.`, maybe?